pypi_formula_mappings.json

```
@@ -317,8 +317,8 @@
     "exclude_packages": ["certifi"]
   },
   "fdroidserver": {
-    "exclude_packages": ["certifi", "cryptography", "numpy", "pillow"],
-    "extra_packages": ["androguard==3.3.5", "setuptools"]
+    "exclude_packages": ["certifi", "cryptography", "numpy", "pillow", "puremagic"],
+    "extra_packages": ["androguard==3.3.5", "biplist", "pycountry", "python-magic", "setuptools"]
```

I'm assuming these extra packages enable optional features? Can fdroidserver define an extra instead so Homebrew (and any `pip` user) doesn't need to manually manage these?

---

Its already in _fdroidserver_:

```python
    extras_require={
        'optional': ['biplist', 'pycountry'],
```

I just don't know how to enable that in the Formula

---

Awesome, so we can do:

```suggestion
    "package_name": "fdroidserver[optional]",   
    "exclude_packages": ["certifi", "cryptography", "numpy", "pillow", "puremagic"],
    "extra_packages": ["androguard==3.3.5", "python-magic", "setuptools"]
```

What do you think about also adding a `libmagic` extra (or similar) that depends on `python-magic`? That way it's more explicit that it's something fdroidserver supports and benefits more than just Homebrew (we try to keep `extra_packages` limited to temporary bugfixes)

---

Thanks, I included your proposal.  Including `python-magic` in `optional` makes sense and is easy.  Dealing with `libmagic` binaries there seems like a pain. I'd prefer to deal with it in the packaging for now at least.